Core/include/Acts/Seeding/SeedFinderOrthogonal.ipp

```diff
@@ -428,6 +429,11 @@ void SeedFinderOrthogonal<external_spacepoint_t>::filterCandidates(
         continue;
       }
 
+      // Protection against div by zero
+      if (B2 == 0) {
+        B2 = std::numeric_limits<float>::min();
+      }
```

Comment:
@andiwand what do you think about this then? 😄

Reply:
hm I think we had a similar fix in the classical seeding. maybe @LuisFelipeCoelho remembers.

I think in this case it is fine to have two cases for field and no field. if `float::min` is a good solution I am not sure because depending on the following calculations you might end up with an over or underflow.

might be worth to have two separate cases for the code below

Reply:
Hi @andiwand in this case `B` is not the magnetic field. 
```cpp
      float A = (lt.V - Vb) / dU;
      float S2 = 1. + A * A;
      float B = Vb - A * Ub;
      float B2 = B * B;
```

Reply:
But indeed in the default seeding we have the following
```cpp
if (B2 == 0) {
          float pTscatterSigma =
              (m_config.highland / m_config.maxPtScattering) *
              m_config.sigmaScattering;
          p2scatterSigma = pTscatterSigma * pTscatterSigma * iSinTheta2;
        } else {
          float pT = options.pTPerHelixRadius * std::sqrt(S2 / B2) / 2.;
          if (pT > m_config.maxPtScattering) {
            float pTscatterSigma =
                (m_config.highland / m_config.maxPtScattering) *
                m_config.sigmaScattering;
            p2scatterSigma = pTscatterSigma * pTscatterSigma * iSinTheta2;
          }
        }
```

So I can try apply the same reasoning here

Reply:
oh thanks for clarifying @CarloVarni. looking at `sqrt(S2)/B = 2 * helixradius` means `B2=0` corresponds to a straight track no?

Reply:
yes indeed

Reply:
ok, replaced (here and in the default seeding) with 
```cpp
if (B2 == 0 or options.pTPerHelixRadius * std::sqrt(S2 / B2) / 2. > m_config.maxPtScattering) {
	  float pTscatterSigma =
	    (m_config.highland / m_config.maxPtScattering) *
	    m_config.sigmaScattering;
          p2scatterSigma = pTscatterSigma * pTscatterSigma * iSinTheta2;
        }
```

Reply:
which should be protect from the division by zero as well